taskt/Core/Automation/Commands/Data Commands/ParseDatasetRowCommand.cs

```diff
@@ -1,101 +1,93 @@
 ﻿using System;
 using System.Collections.Generic;
-using System.Linq;
-using System.Xml.Serialization;
 using System.Data;
+using System.Linq;
 using System.Windows.Forms;
-using taskt.UI.Forms;
-using taskt.UI.CustomControls;
+using System.Xml.Serialization;
+using taskt.Core.Automation.Attributes.ClassAttributes;
+using taskt.Core.Automation.Attributes.PropertyAttributes;
+using taskt.Core.Automation.Engine;
+using taskt.Core.Script;
 using taskt.Core.Utilities.CommonUtilities;
+using taskt.UI.CustomControls;
+using taskt.UI.Forms;
 
 namespace taskt.Core.Automation.Commands
 {
     [Serializable]
-    [Attributes.ClassAttributes.Group("Data Commands")]
-    [Attributes.ClassAttributes.Description("This command allows you to parse a dataset row column into a variable.")]
-    [Attributes.ClassAttributes.UsesDescription("Use this command when you want to extract data from a dataset variable")]
-    [Attributes.ClassAttributes.ImplementationDescription("")]
+    [Group("Data Commands")]
+    [Description("This command parses a dataset row column into a variable.")]
```

Comment:
Do we need this command at all? We already have GetDataRowValueCommand

Reply:
I think GetDataRowValueCommand (one of the DataTable Commands) is already serving the purpose. We can remove ParseDatasetRowCommand from **Data Commands**.